scripts/run-with-gradle-raw-file.sh

```diff
@@ -19,7 +19,7 @@ export KEYCLOAK_SERVICE_ACCOUNT_FILE=/opt/pravega-sensor-collector/conf/keycloak
 export JAVA_OPTS="-Xmx512m"
 
 export PRAVEGA_SENSOR_COLLECTOR_RAW1_CLASS=io.pravega.sensor.collector.rawfile.RawFileIngestService
-export PRAVEGA_SENSOR_COLLECTOR_RAW1_FILE_SPEC=/opt/pravega-sensor-collector/Files
+export PRAVEGA_SENSOR_COLLECTOR_RAW1_FILE_SPEC="/opt/pravega-sensor-collector/Files/A,/opt/pravega-sensor-collector/Files/B"
```

Comment:
Along with Raw file are we adding multiple path for other file types like Parquet and CSV?

Reply:
Yes, I've added this functionality in all 3 services